lib/secure/simul/wiz_map.c

```diff
@@ -0,0 +1,7 @@
+//
```

Comment:
I think we decided not to use this at some point last week?

Reply:
Yep.